docs/en/enable_completion.md

```diff
@@ -94,3 +94,17 @@ pulsarctl completion bash >/usr/local/etc/bash_completion.d/pulsarctl.bash
 echo 'source /usr/local/etc/bash_completion.d/pulsarctl.bash' >> ~/.bashrc
 source ~/.bashrc
 ```
+
+### Fish
+
+To load completions once in your current session run:
+		
+```bash
+pulsarctl completion fish | source
```

Comment:
This command needs to be corrected.

Log:
```
❯ ./pulsarctl completion fish | source                                   
source: not enough arguments

```

Reply:
:thinking: I pushed an update to the PR four days ago, that fixed the issue, but I can have another look.

Reply:
The command succeeds with my `pulsarctl` build from the branch. Are you running the Fish shell when executing `pulsarctl completion fish | source`?

Reply:
> The command succeeds with my `pulsarctl` build from the branch. Are you running the Fish shell when executing `pulsarctl completion fish | source`?

Right.  I'm running this command on zsh.

Reply:
That is probably not supported. I guess you would need to execute this in Fish, if you want to test.

Reply:
Good, it works fine on the fish.

Reply:
Could you check the failed CI?

Reply:
will do

Reply:
I fixed the formatting...